tools/pingtester/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM centos:centos8
```

Comment:
Could we avoid having another image and use a publicly available one, tinkering with the command we are executing? If not, I'd try to squash additional features in the sctptester image so we have only one to maintain.

Reply:
Done